tests/src/Nightwatch/Commands/paragraphs/add.js

```
@@ -18,6 +18,9 @@ exports.command = function add(fieldName, type, position) {
   const browser = this;
 
   if (position < 1) {
+    // Returning of browser without any action breaks test.
+    browser.pause(1);
```

This (and the other instances of browser.pause(1);) hide exceptions. We should handle them more visible to the developer, like logging and exiting, or something similar.